examples/two-step-transfer/Cargo.toml

```diff
@@ -19,7 +19,7 @@ bump_alloc = ["concordium-std/bump_alloc"]
 version = "*"
 path = "../../concordium-std"
 default-features = false
-features = ["concordium-quickcheck"]
+features = ["concordium-quickcheck", "wasm-test"]
```

Comment:
the "wasm-test" feature is a bit special, since it changes how some functions gets compiled and should not be enabled in general, but only for unit test builds.
`cargo-concordium` should already enable this when building for unit testing.